src/oss-find-squats-lib/MutateExtension.cs

```diff
@@ -156,16 +156,23 @@ public static async IAsyncEnumerable<FindPackageSquatResult> EnumerateExistingSq
             {
                 foreach ((string candidatePurlString, IList<Mutation> mutations) in candidateMutations)
                 {
+                    bool useCache = true;
                     if (options?.SleepDelay > 0)
                     {
                         Thread.Sleep(options.SleepDelay);
                     }
+
+                    if (options?.UseCache == false)
+                    {
+                        useCache = false;
+                    }
+
                     // Create the purl from the mutation to see if it exists.
                     PackageURL candidatePurl = new(candidatePurlString);
                     FindPackageSquatResult? res = null;
                     try
                     {
-                        if (await manager.PackageExists(candidatePurl, false))
+                        if (await manager.PackageExists(candidatePurl, useCache))
```

Comment:
Simpler not to have the copy of the use cache variable.
```suggestion
                        if (await manager.PackageExists(candidatePurl, options?.useCache ?? true))
```

Reply:
Also the current implementation actually creates N separate useCache variables, one for every mutation.